x-pack/plugins/actions/common/routes/connector/response/types/v1.ts

```diff
@@ -20,7 +25,11 @@ export interface ConnectorResponse<Config extends ActionTypeConfig = ActionTypeC
   is_preconfigured: ConnectorResponseSchemaType['is_preconfigured'];
   is_deprecated: ConnectorResponseSchemaType['is_deprecated'];
   is_system_action: ConnectorResponseSchemaType['is_system_action'];
-  referenced_by_count: ConnectorResponseSchemaType['referenced_by_count'];
+}
+
+export interface AllConnectorsResponse<Config extends ActionTypeConfig = ActionTypeConfig>
```

Comment:
`AllConnectorsResponse` doesn't quite make sense to me. Maybe `ConnectorWithCountResponse`?

Reply:
It may become misleading if we add another extra param in the Find response.
Actually ConnectorsResponse is enough but could be easily confused with ConnectorResponse.
Maybe we can refactor this later?